editor/export_template_manager.cpp

```diff
@@ -577,7 +581,9 @@ ExportTemplateManager::ExportTemplateManager() {
 	template_downloader = memnew(AcceptDialog);
 	template_downloader->set_title(TTR("Download Templates"));
 	template_downloader->get_ok()->set_text(TTR("Close"));
+	template_downloader->set_exclusive(true);
 	add_child(template_downloader);
+	template_downloader->connect("popup_hide", this, "_window_template_downloader_closed");
```

Comment:
Is this needed now that you've made it exclusive? Why isn't `NOTIFICATION_VISIBILITY_CHANGED` sufficient for that?

Reply:
And sorry for the late answer BTW, post-release days have been really hectic :)

Reply:
Oh, no worries.

Although I made it exclusive, it doesn't provide much of a fix other than to prevent the user from accidentally closing the dialog when the download is in progress.

As for the fix of the url error, the `NOTIFICATION_VISIBILITY_CHANGED` gets called when the manager dialog's state has changed. I may be wrong, but the http request has to be canceled upon closing the downloader dialog to fix this issue.

Reply:
Do you have any suggestions on how it could've been done better?